packages/x-date-pickers-pro/src/DateRangePicker/date-range-manager.test.ts

```diff
@@ -3,9 +3,9 @@ import { calculateRangeChange, calculateRangePreview } from './date-range-manage
 import { adapterToUse } from '../../../../test/utils/pickers-utils';
 import { DateRange } from '../internal/models/dateRange';
 
-const start2018 = adapterToUse.date('2018-01-01T00:00:00.000');
-const mid2018 = adapterToUse.date('2018-06-01T00:00:00.000');
-const end2019 = adapterToUse.date('2019-01-01T00:00:00.000');
+const start2018 = new Date(2018, 0, 1);
```

Comment:
This would only work for adapters that use native `Date` objects (like `date-fns`).
Take `dayjs` for example and its `isAfter` method used by `calculateRangeChange`:

https://github.com/dmtrKovalenko/date-io/blob/77f89b4859d228ed8f34c740e3defc78d52bdba1/packages/dayjs/src/dayjs-utils.ts#L140-L142

It would throw, since `Date` doesn't have `.isAfter` method.

Reply:
I replaced all `new Date(year, month, day)` with `adapterToUse.date(new Date(year, month, day))`. It would be better if we had the option of doing `adapterToUse.date(year, month, day)` but not all adapters support this syntax.

Reply:
Great!